ambry-account/src/main/java/com/github/ambry/account/AccountServiceMetrics.java

```diff
@@ -110,4 +113,24 @@ void trackAccountDataInconsistency(CompositeAccountService compositeAccountServi
     metricRegistry.register(MetricRegistry.name(CompositeAccountService.class, "AccountDataInconsistencyCount"),
         accountDataInconsistencyCount);
   }
+
+  /**
+   * Tracks the number of seconds elapsed since the last database sync.
+   * @param gauge the function returning the elapsed time.
+   */
+  void trackTimeSinceLastSync(Gauge<Integer> gauge) {
+    timeInSecondsSinceLastSyncGauge = gauge;
+    metricRegistry.register(MetricRegistry.name(MySqlAccountService.class, "TimeInSecondsSinceLastSync"),
+        timeInSecondsSinceLastSyncGauge);
+  }
+
+  /**
+   * Tracks the number of seconds elapsed since the last database sync.
```

Comment:
nit: Typo with description.

Reply:
sorry, what's the typo?

Reply:
The description is copied from preivous method, should be updated.

Reply:
oh right, will fix.